### PR TITLE
Update auto-assign-issue action version

### PR DESCRIPTION
**Pull request by fvictorio**
Use the latest commit, which includes a fix that makes the action work with PRs.

### CODE REVIEW

File: .github/workflows/autoassign-issues-and-prs.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - name: "Autoassign issue/PR"
         if: ${{ !contains(fromJson('["alcuadrado","fvictorio","morgansliman","feuGeneA","kanej"]'), github.event.issue.user.login || github.event.pull_request.user.login) }}
-        uses: pozil/auto-assign-issue@v1.1.0
+        uses: pozil/auto-assign-issue@cf0dddf9c33ad096975198b22d05870d8016176c
```
Comment (alcuadrado) [inline .github/workflows/autoassign-issues-and-prs.yml:15]:
```suggestion
        uses: pozil/auto-assign-issue@v1.4.0
```

**Review by alcuadrado — APPROVED** [review]
new version good

**Review by LANCENET — APPROVED** [review]
Lancenet

**Comment by alcuadrado** [comment]
Merged this as PRs are still being submitted and not assigned.

**Pull request merged**